Makefile

```diff
@@ -338,9 +345,17 @@ delete-cluster: $(CLUSTERCTL) ## Deletes the development Kubernetes Cluster "$CL
 	--kubeconfig ./kubeconfig \
 	-p ./examples/_out/provider-components.yaml \
 
+.PHONY: kind-create
+kind-create: ## create capdo kind cluster if needed
+	./scripts/kind-with-registry.sh
+
+.PHONY: delete-cluster
+delete-cluster: delete-workload-cluster  ## Deletes the example kind cluster "capdo"
+	kind delete cluster --name=$(KIND_CLUSTER_NAME)
+
 .PHONY: kind-reset
-kind-reset: ## Destroys the "clusterapi" kind cluster.
-	kind delete cluster --name=clusterapi || true
+kind-reset: ## Destroys the "capdo" kind cluster.
+	kind delete cluster --name=$(KIND_CLUSTER_NAME) || true
```

Comment:
@cpanato should we also remove `kind-registry` container here?
or let it exist and remove it if necessary manually

Reply:
I'm not sure about that, but I'd say let it exist for now and delete if needed.